sci-physics/pythia/pythia-8.2.45-r1.ebuild

```diff
@@ -159,6 +159,7 @@ src_install() {
 	dobin bin/pythia8-config
 	doheader -r include/*
 	dolib.so lib/libpythia8.so
+	use lhapdf && dolib.so lib/libpythia8lhapdf6.so
```

Comment:
This is going to require a revbump in order to propagate this change to all users.